spec/system/admin_customize_themes_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+describe 'Admin Customize Themes', type: :system, js: true do
+  fab!(:color_scheme) { Fabricate(:color_scheme) }
+  fab!(:theme) { Fabricate(:theme) }
+  fab!(:admin) { Fabricate(:admin) }
+
+  before do
+    sign_in(admin)
+  end
+
+  describe 'when visiting the page to customize the theme' do
+    it 'should allow admin to update the color scheme of the theme' do
+      visit("/admin/customize/themes/#{theme.id}")
+
+      within(".theme-settings__color-scheme") do
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
+
+        find(".color-palettes").click
+        find(".color-palettes-row[data-value='#{color_scheme.id}']").click
+        find(".submit-edit").click
+
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
```

Comment:
This is slightly risky test which could stop testing what you think it is very easily.

If for some reason you don't end up on the page you think you should be after `find(".submit-edit").click` your test will still green.

Also Im not used to use `within()` but shouldn't we have these two expects outside of it?

Reply:
Good point :+1: